sonar-db/src/test/java/org/sonar/db/component/ComponentDaoTest.java

```diff
@@ -560,6 +567,7 @@ public void insert() {
 
     ComponentDto componentDto = new ComponentDto()
       .setUuid("GHIJ")
+      .setUuidPath("ABCD.EFGH.GHIJ.")
```

Comment:
no uuid path can take that form (it should start with a `.`)
if you don't care about the value, put `uuid_path` , don't falsly mimic the real content of the column, it will confuse people looking at the text later one
(the fact that the same comment applies to old code doesn't make it an excuse to keep on following the bad practice)